### PR TITLE
Adds TypeScript to list of implementations.

### PR DESCRIPTION
**Pull request by queercat**
# Description

This just adds a link to my repository with an implementation of a Pratt parser for Bantam.

# Summary of changes
- Updates README with link to myself.
- Adds a comment to the README regarding future consideration of contributor list separation.

# Garfield comic (because I like them.)
blog_image_4370_10208_Garfield_By_Jim_Davis_Oct_3__2011_201711011535

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,12 @@ The code here has also been ported to:
 - [C++][cpp] and [Rust][rust] by Jürgen Wurzer,
 - [Python][python] by Kuba Sunderland-Ober.
 - [MiniScript][miniscript] by Joe Strout.
+- [TypeScript][typescript] by May Tusek. <!-- Not sure if this should be a comma or period, IMO refactor all to periods. -->
```
Comment (munificent) [inline README.md:12]: If you'd like, probably the best answer is no punctuation at all. If you'd like to do that clean-up, I'd appreciate it. If not, that's OK. :)

**Review by munificent — CHANGES_REQUESTED** [review]
If you want to clean up the punctuation, that's fine. If not, just leave it as is and go ahead and remove the `<!-- ... -->` comment.

Thanks for the link!

File: README.md (hunk 1 above)
Comment (queercat) [reply README.md]: Punctuation has been removed :)

Comment (munificent) [reply README.md]: Thank you!

**Pull request merged**